rootston/config.c

```diff
@@ -466,11 +470,18 @@ struct roots_config *roots_config_create_from_args(int argc, char *argv[]) {
 		case 'D':
 			config->debug_damage_tracking = true;
 			break;
+		case 'l':
+			log_verbosity = strtoul(optarg, NULL, 10);
+			if (log_verbosity >= WLR_LOG_IMPORTANCE_LAST) {
+				log_verbosity = WLR_LOG_IMPORTANCE_LAST - 1;
+			}
+			break;
 		case 'h':
 		case '?':
 			usage(argv[0], c != 'h');
 		}
 	}
+	wlr_log_init(log_verbosity, NULL);
```

Comment:
Hmm. Now `wlr_log_init` is called twice. Do we care? We should probably update the wlr-log docs to allow this.

Reply:
The function doesn't really care how many times it's called, so I let it be. It acts kind of like an "upgrade" of the log level (from DEBUG to whatever is given).

Reply:
I don't care if we call it several times

Reply:
Can you add this to the docs?

Reply:
Sorry, I missed this earlier. Added a short blurb in the header.